rgw/v2/tests/s3cmd/test_header_size.py

```diff
@@ -0,0 +1,148 @@
+"""
+test_header_size - Test to verify header size in Beast front end using s3cmd
+
+Usage: test_header_size.py -c <input_yaml>
+
+<input_yaml>
+    Note: Following yaml can be used
+    test_header_size.yaml
+
+Operation:
+    Create an user
+    Create a bucket with user credentials
+    Performs object upload with header
+    Verify object for its header
+"""
+
+
+import argparse
+import json
+import logging
+import os
+import random
+import sys
+import traceback
+from string import ascii_lowercase, ascii_uppercase, digits
+
+sys.path.append(os.path.abspath(os.path.join(__file__, "../../../..")))
+
+
+from v2.lib import resource_op
+from v2.lib.exceptions import RGWBaseException, TestExecError
+from v2.lib.s3.write_io_info import BasicIOInfoStructure, IOInfoInitialize
+from v2.lib.s3cmd import auth as s3_auth
+from v2.tests.s3cmd import reusable as s3cmd_reusable
+from v2.utils import utils
+from v2.utils.log import configure_logging
+from v2.utils.test_desc import AddTestInfo
+
+log = logging.getLogger()
```

Comment:
could you add the filename... this helps during triage... `log = logging.getLogger(__name__)`